oasislmf/computation/hooks/post_analysis.py

```diff
@@ -1,16 +1,48 @@
-# TO DO -- placeholder for post analysis output modification hook
-
-# __all__ = [
-#    '_'
-# ]
-#
-#
-# class _(ComputationStep):
-#    """
-#    - desc here -
-#    """
-#    step_params = [  .. args ..  ]
-#
-#
-#    def run(self):
-#        return None
+__all__ = [
+    'PostAnalysis'
+]
+
+import os
+
+from ..base import ComputationStep
+from ...utils.path import get_custom_module
+from ...utils.exceptions import OasisException
+
+
+class PostAnalysis(ComputationStep):
+    """Computation step that is called after loss calculations.
+
+    It passes the output directory to a customisable function that might modify or add to the
+    standard output files.
+    """
+    step_params = [
+        {'name': 'post_analysis_module', 'required': True, 'is_path': True, 'pre_exist': True,
```

Comment:
Can we easily support a Python dotted module path instead of a file system path? I think it would make configuration easier.